package.json

```diff
@@ -102,7 +102,7 @@
 		"prepack": "rm -rf lib && tsc -b && oclif-dev manifest && oclif-dev readme",
 		"postpack": "rm -f oclif.manifest.json",
 		"lint": "npx eslint ./src --ext .ts",
-		"test": "./gradlew test && npx nyc mocha \"./test/**/*.test.ts\"",
+		"test": "./gradlew test && npx --always-spawn nyc mocha --timeout 10000 --retries 5 \"./test/**/*.test.ts\"",
```

Comment:
This is the change that works around the issue. All other changes are things I came across while debugging the issue.